### DISCUSSION TITLE
Adjusting width of columns

### DISCUSSION THREAD

**Discussion by dwt1978** [rejetto/hfs#201] [General]
I'm using HFS 2.3m. How do I increase the width of the columns. I want to increase the width of the column "Name .extension" on all pages. Thanks.

**Comment by rejetto** [comment]
sorry, this place is only for HFS 3.
since i'm not using windows anymore, I don't have hfs 2 at hand to check.
you'll find more help by the guys on the forum
https://rejetto.com/forum/index.php?board=21.0
